[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_filter_get_message_after_node2_restarts with id: 2024-12-04_11-44-46__38552f8f-a030-44c7-9f77-6a82f9fef3b6
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.filter:filter.py:28 Running fixture setup: filter_setup
[35mDEBUG   [0m src.steps.filter:filter.py:34 Running fixture setup: setup_main_relay_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node1_2024-12-04_11-44-46__38552f8f-a030-44c7-9f77-6a82f9fef3b6__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:33 Network waku created
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.196.127
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['49631', '49632', '49633', '49634', '49635']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 49631:49631 -p 49632:49632 -p 49633:49633 -p 49634:49634 -p 49635:49635 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=49633 --rest-port=49631 --tcp-port=49632 --discv5-udp-port=49634 --rest-address=0.0.0.0 --nat=extip:172.18.196.127 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=64146fd3fe9c7562af500ccfdaad9c35b1a2db1e792c1cfadf0f9af98b158034 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=49635 --metrics-logging=true --relay=true --filter=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.196.127 waku 3218fad3c14a58a5b4180b511513a389d59db85bc16f9e5361c4f61892abb132
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 3218fad3c14a. Setting up logs at ./log/docker/node1_2024-12-04_11-44-46__38552f8f-a030-44c7-9f77-6a82f9fef3b6__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/nwaku:latest. REST: 49631
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:49631/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:49631/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.196.127/tcp/49632/p2p/16Uiu2HAm6mmY2HoGgJqUYiSrpYcrQL5SWMNDhmxqNscmLM6Uo8eH","/ip4/172.18.196.127/tcp/49633/ws/p2p/16Uiu2HAm6mmY2HoGgJqUYiSrpYcrQL5SWMNDhmxqNscmLM6Uo8eH"],"enrUri":"enr:-L24QN3b3sZtG2EgztUuYBc8VOQuXdYrxeYVpwi3GYgTk2YVCPqSevHmSKp94MOj0fFu26aMKU7cJKFKrDYD-rxGkbgCgmlkgnY0gmlwhKwSxH-KbXVsdGlhZGRyc5YACASsEsR_BsHgAAoErBLEfwbB4d0DgnJzhQADAQAAiXNlY3AyNTZrMaECqJZ0Rn3TXrfu-ckW29eByg5EUuF7ydmMRYGfUNs1Uu6DdGNwgsHgg3VkcILB4oV3YWt1MgU"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[35mDEBUG   [0m src.steps.filter:filter.py:39 Running fixture setup: setup_main_filter_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node2_2024-12-04_11-44-46__38552f8f-a030-44c7-9f77-6a82f9fef3b6__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.219.60
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['21777', '21778', '21779', '21780', '21781']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 21777:21777 -p 21778:21778 -p 21779:21779 -p 21780:21780 -p 21781:21781 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=21779 --rest-port=21777 --tcp-port=21778 --discv5-udp-port=21780 --rest-address=0.0.0.0 --nat=extip:172.18.219.60 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=d7af85bc23b8ded2b68db2f7ae7dd7d2fedf3c2a00d1c8d0ecd928bedd98110c --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=21781 --metrics-logging=true --relay=false --discv5-bootstrap-node=enr:-L24QN3b3sZtG2EgztUuYBc8VOQuXdYrxeYVpwi3GYgTk2YVCPqSevHmSKp94MOj0fFu26aMKU7cJKFKrDYD-rxGkbgCgmlkgnY0gmlwhKwSxH-KbXVsdGlhZGRyc5YACASsEsR_BsHgAAoErBLEfwbB4d0DgnJzhQADAQAAiXNlY3AyNTZrMaECqJZ0Rn3TXrfu-ckW29eByg5EUuF7ydmMRYGfUNs1Uu6DdGNwgsHgg3VkcILB4oV3YWt1MgU --filternode=/ip4/172.18.196.127/tcp/49632/p2p/16Uiu2HAm6mmY2HoGgJqUYiSrpYcrQL5SWMNDhmxqNscmLM6Uo8eH
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.219.60 waku b13eb4c23c2ba636542185eb6f1aef3229def2c313b783883aa69cb81336992c
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID b13eb4c23c2b. Setting up logs at ./log/docker/node2_2024-12-04_11-44-46__38552f8f-a030-44c7-9f77-6a82f9fef3b6__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/nwaku:latest. REST: 21777
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:21777/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:21777/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.219.60/tcp/21778/p2p/16Uiu2HAmNruxHbGfxZ5i8yvoPfhLSgsPpKk1fAf4gdNizpa2zUJA","/ip4/172.18.219.60/tcp/21779/ws/p2p/16Uiu2HAmNruxHbGfxZ5i8yvoPfhLSgsPpKk1fAf4gdNizpa2zUJA"],"enrUri":"enr:-L24QODQumvQM5hLP1RDsRqNInVtJd82oi8Zy8nD4JAepOOACMSSaHOtG-B0ZOs8vdV69bBrrTIkWrt09yO3UVElTzoCgmlkgnY0gmlwhKwS2zyKbXVsdGlhZGRyc5YACASsEts8BlUSAAoErBLbPAZVE90DgnJzhQADAQAAiXNlY3AyNTZrMaEDl6LeIrEjiDtwz4IwZBrlQef1Ih75CQ_xFwH5676Uz4eDdGNwglUSg3VkcIJVFIV3YWt1MgA"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:21777/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.196.127/tcp/49632/p2p/16Uiu2HAm6mmY2HoGgJqUYiSrpYcrQL5SWMNDhmxqNscmLM6Uo8eH"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.filter:filter.py:47 Running fixture setup: subscribe_main_nodes
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:49631/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:21777/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "a31d0104-c11c-46a4-be72-a849cd1d67dc", "contentFilters": ["/test/1/waku-filter/proto"], "pubsubTopic": "/waku/2/rs/3/1"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"a31d0104-c11c-46a4-be72-a849cd1d67dc","statusDesc":"OK"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:49631/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "RmlsdGVyIHdvcmtzISE=", "contentTopic": "/test/1/waku-filter/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.filter:filter.py:98 Checking that peer NODE_2:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:21777/filter/v2/messages/%2Ftest%2F1%2Fwaku-filter%2Fproto" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"RmlsdGVyIHdvcmtzISE=","contentTopic":"/test/1/waku-filter/proto","version":0,"timestamp":1733312706196133376,"ephemeral":false}]'
[35mDEBUG   [0m src.node.waku_node:waku_node.py:249 Restarting container with id b13eb4c23c2b
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:21777/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:21777/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.219.60/tcp/21778/p2p/16Uiu2HAmNruxHbGfxZ5i8yvoPfhLSgsPpKk1fAf4gdNizpa2zUJA","/ip4/172.18.219.60/tcp/21779/ws/p2p/16Uiu2HAmNruxHbGfxZ5i8yvoPfhLSgsPpKk1fAf4gdNizpa2zUJA"],"enrUri":"enr:-L24QODQumvQM5hLP1RDsRqNInVtJd82oi8Zy8nD4JAepOOACMSSaHOtG-B0ZOs8vdV69bBrrTIkWrt09yO3UVElTzoCgmlkgnY0gmlwhKwS2zyKbXVsdGlhZGRyc5YACASsEts8BlUSAAoErBLbPAZVE90DgnJzhQADAQAAiXNlY3AyNTZrMaEDl6LeIrEjiDtwz4IwZBrlQef1Ih75CQ_xFwH5676Uz4eDdGNwglUSg3VkcIJVFIV3YWt1MgA"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 2 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:49631/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:21777/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "8015687c-d3ee-4de1-8654-b4d66d737fcb", "contentFilters": ["/test/1/waku-filter/proto"], "pubsubTopic": "/waku/2/rs/3/1"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"8015687c-d3ee-4de1-8654-b4d66d737fcb","statusDesc":"OK"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:49631/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "RmlsdGVyIHdvcmtzISE=", "contentTopic": "/test/1/waku-filter/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.filter:filter.py:98 Checking that peer NODE_2:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:21777/filter/v2/messages/%2Ftest%2F1%2Fwaku-filter%2Fproto" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"RmlsdGVyIHdvcmtzISE=","contentTopic":"/test/1/waku-filter/proto","version":0,"timestamp":1733312739833186560,"ephemeral":false}]'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id 3218fad3c14a
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id b13eb4c23c2b
[1m[31mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 3218fad3c14a. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.